pkg/server/endpoints/config.go

```diff
@@ -61,7 +62,12 @@ type Config struct {
 	// Makes policy decisions
 	AuthPolicyEngine *authpolicy.Engine
 
-	Log     logrus.FieldLogger
+	// The fielLogger interface
+	Log             logrus.FieldLogger
```

Comment:
I'd suggest changing this to a *logrus.Logger and threading it through to the logger service (see my comment in the logger service implementation)

Reply:
You can't make it a logger instance.  It will lack the API fields that the product uses.

Reply:
Resolved in latest.